packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -38,6 +38,9 @@ const allowedErrors = {
   NetworkError: 'NetworkError when attempting to fetch resource.',
   // Webkit browser error
   CORSError: 'Cross-origin redirection',
+
+  //Prompt error
+  PromptError: 'User is not authenticated or session cookie not sent to the server',
```

Comment:
I think the `error` parameter may be better than the `error_description` parameter. Rather than this long sentence, you'd look for this: `interaction_required`.

Reply:
Actually, looking at this now, the most minimal code change would be to use both the `error` and `error_description` as defined by this `parseError` method: https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/packages/javascript-sdk/src/oauth2-client/index.ts#L293.